Contributors (#1742)
modelica/fmi-standard: docs/7___appendix.adoc

- Dassault Syst&#232;mes (Sweden), Link&#246;ping University and Modelon AB thank the Swedish funding agency VINNOVA (2012--01157) for partial funding of this work within MODRIO.

- Siemens PLM Software (France) and IFPEN thank DGCIS for partial funding of this work within MODRIO.

[appendix]
== Contributors

The development group for FMI 3.0 was headed by Andreas Junghanns (Synopsys). 

Special thanks to 

- Andreas Junghanns (Synopsys) and Torsten Sommer (Dassault Systèmes) for the initial conversion of the FMI 2.0 Word document to ASCIIDOC

> -  Torsten Sommer (Dassault Systèmes)  for the creation of the reference FMUS

Reviewer: ```suggestion
- Torsten Sommer (Dassault Systèmes)  for the creation of the reference FMUS
```

> -  Cláudio Gomes (Aarhus University) for the creation of the tooling to create the graphical representation of the XML schemas.

Reviewer: ```suggestion
- Cláudio Gomes (Aarhus University) for the creation of the tooling to create the graphical representation of the XML schemas.
```

> - Andreas Junghanns (Synopsys) ans Torsten Blochwitz (ESI-ITI) for restructuring and re-writing the whole FMI standard text.

Reviewer: ```suggestion
- Andreas Junghanns (Synopsys) and Torsten Blochwitz (ESI-ITI) for restructuring and re-writing the whole FMI standard text.
```

> - all companies and organizations that participated in the FMI 3.0 Plugfests: Aarhus University, Altair, Augsburg University, AVL, Dassault Systèmes, dSPACE, ESI-ITI, ETAS, Julia Computing, MaplesSoft, OpenModelica, PMSF IT Consulting,  Synopsys, TLK

Reviewer 2: ```suggestion
- all companies and organizations that participated in the FMI 3.0 Plugfests: Aarhus University, Altair, Augsburg University, AVL, Dassault Systèmes, dSPACE, ESI-ITI, ETAS, Julia Computing, Maplesoft, OSMC, PMSF IT Consulting, Synopsys, TLK
```

The essential part of the design of this version was performed by (alphabetical list)

Christian Bertsch, Robert Bosch GmbH, Germany
Matthias Blesken, dSPACE GmbH, Germany

> Torsten Blochwitz, ITI, Germany

Reviewer 2: ```suggestion
Torsten Blochwitz, ESI-ITI, Germany
```

Cláudio Gomes, Aarhus University, Denmark
Andreas Junghanns, Synopsys, Germany
Pierre R. Mai, PMSF IT Consulting, Germany
Masoud Najafi, Altair, France
Andreas Pillekeit, dSPACE GmbH, Germany
Klaus Schuch, AVL List GmbH, Austria
Christian Schulze, TLK-Thermo GmbH, Germany
Torsten Sommer, Dassault Systèmes, Germany
Karl Wernersson, Dassault Systèmes, Sweden
Irina Zacharias, dSPACE GmbH, Germany

Additionally the following partners participated at FMI 3.0 design meetings and contributed to the discussion (alphabetical list)

Nick Battle, United Kingdom
Thomas Beutlich, Germany
Jorge Bernal-Romero, ITK-Engineering, Germany

> Robert Braun, Linköping Uniersity, Sweden

Reviewer 2: ```suggestion
Robert Braun, Linköping University, Sweden
```

Rüdiger Franke, ABB AG, Germany
Markus Friedrich, SIMPACK AG, Germany
Jan Niklas Jaeschke, TLK-Thermo, Germany
Oliver Kotte, Robert Bosch GmbH, Germany

> Kaska Kowalalska, Maplessoft, Canada

Reviewer 2: ```suggestion
Kaska Kowalalska, Maplesoft, Canada
```

Timo Penndorf, ETAS GmbH, Germany
Tim Schenk, Siemens AG, Germnay

> Patrick Täuber, DSPACE, Germany

Reviewer 2: ```suggestion
Patrick Täuber, dSPACE GmbH, Germany
```

Adrian Tiera, Synopsys, Romania

> Otto Tronarp, Wolfram, Sweden

Reviewer 3: ```suggestion
Otto Tronarp, Wolfram MathCore, Sweden
```
